src/app/_components/ImageWithFallback.tsx

```diff
@@ -0,0 +1,68 @@
+import type { CSSProperties } from 'react'
+
+import type { StaticImport } from 'next/dist/shared/lib/get-img-props'
+import Image from 'next/image'
+import type { ImageProps } from 'next/image'
+
+import { graphicsData } from '@/data/graphicsData'
+
+type ImageWithFallbackProps = ImageProps & {
+  fallbackStyle?: CSSProperties
+}
+
+const { src: fallbackSrc, alt: fallbackAlt } = graphicsData.imageFallback
+
+async function getImageSrc(src: string | StaticImport) {
+  const isStaticImport = typeof src === 'object'
```

Comment:
I would probably rename everything to use `static` or `dynamic` since those terms come straight from Next.js docs. Is there an example where this wouldn't work?